airflow/api_connexion/endpoints/log_endpoint.py

```diff
@@ -14,13 +14,99 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import logging
+import os
+from io import BytesIO
 
-# TODO(mik-laj): We have to implement it.
-#     Do you want to help? Please look at: https://github.com/apache/airflow/issues/8135
+from flask import Response, current_app, request
+from itsdangerous.exc import BadSignature
+from itsdangerous.url_safe import URLSafeSerializer
 
+from airflow import models, settings
+from airflow.api_connexion.exceptions import BadRequest, NotFound
+from airflow.configuration import conf
+from airflow.models import DagRun
+from airflow.settings import STORE_SERIALIZED_DAGS
+from airflow.utils.helpers import render_log_filename
+from airflow.utils.session import provide_session
 
-def get_log():
+if os.environ.get('SKIP_DAGS_PARSING') != 'True':
+    dagbag = models.DagBag(settings.DAGS_FOLDER, store_serialized_dags=STORE_SERIALIZED_DAGS)
+else:
+    dagbag = models.DagBag(os.devnull, include_examples=False)
```

Comment:
We need to come up with a way to make DagBag created only one for the webserver. Web and API should share one instance.

Reply:
Should I use the dagbag created at airflow/www/views.py?

Reply:
I rather thought about adding a new field in flask_app or adding to the context.  This is a slightly larger and more problematic change.
https://flask.palletsprojects.com/en/1.1.x/appcontext/

Reply:
I am looking at this now, I think a separate PR is necessary?